weechat_otr_test/test_command_finish.py

```diff
@@ -21,23 +21,23 @@ def after_setup(self):
         weechat_otr.ACCOUNTS['nick@server'] = account
 
     def test_finish_buffer(self):
-        weechat_otr.command_cb(None, 'server_nick2_buffer', 'finish')
+        self.call_command_cb(None, 'server_nick2_buffer', 'finish')
```

Comment:
Of course, that is the correct behavior. I'm just looking for a way to not have 100+ (if I counted correctly) calls to `weechat_otr.PYVER.to_str` in the tests which are easy to forget 😏 

How would you feel about importing a derived class of `weechat_otr` in the tests that only contains all the `*_cb` methods which convert the strings to bytes and calls the parent method?
But if you just want all the strings converted in the argument lists, feel free to say so.